src/services/ltiTool/model.js

```diff
@@ -37,6 +37,7 @@ const ltiToolSchema = new Schema({
 	oAuthClientId: { type: String },
 	friendlyUrl: { type: String, unique: true, sparse: true },
 	skipConsent: { type: Boolean },
+	openNewTab: { type: Boolean, default: false },
```

Comment:
```suggestion
	openNewTab: { type: Boolean, default: false, required:true },
```
and add a migration which sets this default. otherwise, add a helper function that is used for this check which handles isOpenNewTab(model):boolean